src/Entity/Transaction.php

```diff
@@ -99,22 +88,24 @@ class Transaction
      */
     private $donation;
 
-    public function __construct(Donation $donation, array $payboxPayload)
+    public function __construct(Donation $donation, ?array $payboxPayload, ?\DateTimeInterface $donatedAt = null)
```

Comment:
can we avoid changing the construct of Transaction if you don't use anymore Transaction for Transfer/Check payment?